.github/workflows/update_disabled_tests.yml

```diff
@@ -27,8 +27,10 @@ jobs:
           # The token is used to confirm issue creator's write permission before
           # allowing them to disable jobs. Note that the token needs to have access
           # to the target repo, i.e. pytorch/pytorch instead of test-infra. Using
-          # PyTorch bot token is the most obvious choice.
-          GH_PYTORCHBOT_TOKEN: ${{ secrets.GH_PYTORCHBOT_TOKEN }}
+          # PyTorch bot token is the most obvious choice.  Outside of the
+          # environment, we do not have access to this token so fall back to the
+          # GITHUB_TOKEN.
+          GITHUB_TOKEN: ${{ github.ref == 'refs/heads/main' && secrets.GH_MERGEBOT_TOKEN || secrets.GITHUB_TOKEN }}
```

Comment:
wait lol what is the diff between GH_MERGEBOT_TOKEN and GH_PYTORCHBOT_TOKEN

Reply:
I assume one is from pytorchmergebot and the other from pytorchbot but I don't actually know

I renamed here to GITHUB_TOKEN because I thought most people have a personal token saved as GITHUB_TOKEN and no mergebot token so when running locally they wouldn't have to remap.  Also I guess the name would be misleading when the workflow is just using github token